src/nvme/types.h

```diff
@@ -6276,7 +6283,9 @@ enum nvme_status_field {
 	NVME_SC_CMD_INTERRUPTED			= 0x21,
 	NVME_SC_TRAN_TPORT_ERROR		= 0x22,
 	NVME_SC_PROHIBITED_BY_CMD_AND_FEAT	= 0x23,
-	NVME_SC_ADMIN_CMD_MEDIA_NOT_READY	= 0x24,
+	NVME_SC_FDP_DISABLED				= 0x24,
+	NVME_SC_INVALID_PLACEMENT_HANDLE_LIST	= 0x29,
+	NVME_SC_ADMIN_CMD_MEDIA_NOT_READY	= 0x2A,
```

Comment:
These values look wrong. The copy of TP4146 says

- FDP Disabled: 0x29
- Invalid Placement Handle List: 0x2a

And the base spec says for 

- Admin Command Media Not Ready: 0x24

Reply:
mistake(I've pushed wrong branch that was first version of this.) :( 
thanks for the checking, !!

fixed